pkg/volume/scaleio/sio_volume.go

```diff
@@ -265,19 +265,15 @@ func (v *sioVolume) Provision(selectedNode *api.Node, allowedTopologies []api.To
 
 	// setup volume attrributes
 	genName := v.generateName("k8svol", 11)
-	var oneGig int64 = 1024 * 1024 * 1024
-	var eightGig int64 = 8 * oneGig
 
 	capacity := v.options.PVC.Spec.Resources.Requests[api.ResourceName(api.ResourceStorage)]
-	volSizeBytes := capacity.Value()
-	volSizeGB := int64(util.RoundUpSize(volSizeBytes, oneGig))
-
-	if volSizeBytes == 0 {
-		return nil, fmt.Errorf("invalid volume size of 0 specified")
+	volSizeGB, err := util.RoundUpToGiB(capacity)
+	if err != nil {
+		return nil, err
 	}
 
-	if volSizeBytes < eightGig {
-		volSizeGB = int64(util.RoundUpSize(eightGig, oneGig))
+	if volSizeGB < 8 {
```

Comment:
should we return error when volSizeGB == 0?
```
	if volSizeGB == 0 {	
		return nil, fmt.Errorf("invalid volume size of 0 specified")	
	}
```

Reply:
the check is unnecessary because kubernetes will validate it when the pvc is created
```
The PersistentVolumeClaim "myclaim" is invalid: spec.resources[storage]: Invalid value: "0": must be greater than zero
```